docs/data/material/migration/migrating-to-v6/migrating-to-pigment-css.md

```diff
@@ -76,7 +76,9 @@ const nextConfig = {
 /**
  * @type {import('@pigment-css/nextjs-plugin').PigmentOptions}
  */
-const pigmentConfig = {}; // we will refer to this later
+const pigmentConfig = {
+  transformLibraries: ['@mui/material'],
+};
```

Comment:
Some of these seem to have leaked from https://github.com/mui/material-ui/pull/43217?